bodhi/client/__init__.py

```diff
@@ -162,14 +162,21 @@ def edit(user, password, url, **kwargs):
     client = bindings.BodhiClient(base_url=url, username=user, password=password,
                                   staging=kwargs['staging'])
 
+    if kwargs['notes_file'] is not None:
+        if kwargs['notes'] is None:
+            with open(kwargs['notes_file'], 'r') as fin:
+                kwargs['notes'] = fin.read()
+        else:
+            click.echo("ERROR: Cannot specify --notes and --notes-file")
+            sys.exit(1)
```

Comment:
This same code block appears in the ```new()``` function above. It would be good to make it a new function and have both ```new()``` and ```edit()``` use it.